rnd/autogpt_builder/src/components/edit/control/BlocksControl.tsx

```diff
@@ -41,9 +44,15 @@ export const BlocksControl: React.FC<BlocksControlProps> = ({
   );
 
   return (
-    <Popover>
+    <Popover open={pinBlocks ? true : undefined}>
```

Comment:
??? Why undefined?

Reply:
`open=true/false` will force `Popover` to be always opened/closed. `undefined` will make it behave as usual - when user clicks on the button it'll open when user clicks away it'll be hidden.